charts/brig/templates/deployment.yaml

```diff
@@ -167,3 +178,14 @@ spec:
               port: {{ .Values.service.internalPort }}
           resources:
 {{ toYaml .Values.resources | indent 12 }}
+  {{- if .Values.profLogs.enabled }}
+  volumeClaimTemplates:
+    - metadata:
+        name: prof-logs
+      spec:
+        accessModes: [ "ReadWriteOnce" ]
+        storageClassName: {{ .Values.profLogs.storageClassName }}
+        resources:
+          requests:
+            storage: "10Gi"
```

Comment:
I'm wondering if that's enough. I got close to 9gb in 12-13 hours of profiling. Maybe we do `50gi`?

Reply:
no. None of the available tools can handle event logs of > 2G graciously. If we go farther above we won’t get anything useful out of it. That’s why we increase the sample interval from .1 to 1 seconds

Reply:
Also your profile didn’t do the right thing, unfortunately, the ones that we need produce ~2GB per hour.